src/services/rooms.service.js

```diff
@@ -0,0 +1,24 @@
+const { Room } = require('../models/room.model');
+
+const create = async (room) => {
+  return Room.create(room);
+};
+
+const getAll = async () => {
+  return Room.findAll();
+};
+
+const update = async (room) => {
+  return Room.update({ ...room }, { where: { id: room.id } });
```

Comment:
Consider adding validation for the `room` object to ensure it meets the expected format and constraints before attempting to update a room.